apex_launchtest/examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Examples
+
+## `good_proc.test.py`
+
+Usage:
+> apex_launchtest examples/good_proc.test.py
+
+This test launches a python process that spins forever waiting for ctrl+c
+to shut it down.
```

Comment:
It shuts down by itself after ~5s and does not spin forever.

Reply:
I'll clarify the documentation here, but what it's saying is correct.  The test runs for 5 seconds because it stops the process once it sees "Loop 4" printed but the process being tested works as described.

Reply:
@dejanpan done - put more language to disambiguate the difference between what the test does, and what the process being tested does